CMakeLists.txt

```diff
@@ -82,6 +93,7 @@ if (POLICY CMP0042)
   if (APPLE AND NOT DEFINED CMAKE_MACOSX_RPATH)
     set(CMAKE_SKIP_BUILD_RPATH FALSE)
     set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+    # TODO: Verify that this is indeed correct.
```

Comment:
Have you done so? 🙂

Reply:
This needs to be done by someone that truely understands how the dynamic loading process on macOS works. I haven't been able to make to connect the dots between the various sources of information on this.

Reply:
Fair enough. Given that macOS is predominantly a developer platform, we can move on.